src/test/data/JsonSerializableEventManagerTest/duplicateEventEventManager.json

```diff
@@ -0,0 +1,98 @@
+{
+  "events": [
+    {
+      "name": "Invalid Event",
+      "attendees": [
+        {
+          "name": "Alice Pauline",
```

Comment:
Maybe storing the person's UUID instead of the entire object of name, email, address etc might be better? Since that's why we came up with the unique ID for person object in the first place. But this works too!

Reply:
Good point, I will implement this to the next iteration so that we can reduce data stored in EventManagerStorage

Reply:
Yup I agree, this might lead to data inconsistencies, but we can change this in a separate issue